network/dag/publisher.go

```diff
@@ -62,6 +62,7 @@ func (s *replayingDAGPublisher) ConfigureCallbacks(state State) {
 		if payload != nil {
 			s.payloadWritten(ctx, transaction, payload)
 		}
+		return nil
```

Comment:
would it make sense to now return the errors that can occur in payloadWritten (but are just logged)